packages/hyperion-autologging/src/ALType.ts

```diff
@@ -43,4 +43,9 @@ export type ALReactElementEvent = Readonly<{
 export type ALSharedInitOptions = Types.Options<{
   flowletManager: ALFlowletManager;
   domSurfaceAttributeName: string;
-}>;
\ No newline at end of file
+}>;
+
+export type ALElementNameResult = Readonly<{
+  text: string,
+  source: 'innerText' | 'aria-label' | 'aria-labelledby';
+}>;
```

Comment:
Move this type to packages/hyperion-autologging/src/ALInteractableDOMElement.ts. No one else needs it.
In general, it is better to keep the types as close to the place they are produced and avoid one giant file with all the types. Imagine if one day we remove packages/hyperion-autologging/src/ALInteractableDOMElement.ts, and replace with something new, then we want to have the least amount of places to clean up. That's usually a good test of where to put types. 
packages/hyperion-autologging/src/ALType.ts is good for types that are global and needed in many places.